packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -632,6 +633,9 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
       actions.push({
         title: 'Copy Cell',
         group: IrisGridContextMenuHandler.GROUP_COPY,
+        shortcutText: ContextActionUtils.isMacPlatform()
```

Comment:
I was debating if we should add support for Shortcut to handle mouse clicks... would be kind of interesting in that they could be configurable, but it's more of a headache than we need right now.